account-kit/rn-signer/src/client.ts

```diff
@@ -58,6 +76,12 @@ export class RNSignerClient extends BaseSignerClient<undefined> {
       connection,
     });
 
+    this.webauthnStamper = new PasskeyStamper({
+      rpId: rp?.id ?? "localhost",
+    });
+    this.rpId = rp?.id ?? "localhost";
+    this.rpName = rp?.name ?? "localhost";
```

Comment:
if we don't get the `rp` config I think we actually need to not setup the stamper, and then below we need to throw if there's no config for the stamper